src/components/common/NavBar.js

```diff
@@ -3,18 +3,19 @@ import React from 'react';
 /**
  * Required Props:
  * brandTitle {string} - String representing the website brand name
+ * toggleSidebar {function} - Callback for toggling the sidebar
  * TODO: Include other nav bar links
  */
 function NavBar(props) {
-  const { brandTitle } = props;
+  const { brandTitle, toggleSidebar } = props;
 
   return (
     <div className="nav-bar-container">
       <div className="nav-back-btn">
```

Comment:
Could we actually give this entire div the onClick functionality, so both the text and the hamburger icon can toggle the sidebar?